app/src/main/java/com/github/dfa/diaspora_android/task/GetPodsService.java

```diff
@@ -73,24 +71,20 @@ private void getPods() {
                 // TODO: Update deprecated code
 
                 StringBuilder builder = new StringBuilder();
-                HttpClient client = new DefaultHttpClient();
+                //HttpClient client = new DefaultHttpClient();
                 List<String> list = null;
                 try {
-                    HttpGet httpGet = new HttpGet("http://podupti.me/api.php?key=4r45tg&format=json");
-                    HttpResponse response = client.execute(httpGet);
-                    StatusLine statusLine = response.getStatusLine();
-                    int statusCode = statusLine.getStatusCode();
+                    HttpsURLConnection connection = NetCipher.getHttpsURLConnection("https://podupti.me/api.php?key=4r45tg&format=json");
```

Comment:
Connection not disconnected and leaves inputStream open.
Define InputStream and connection above try with null;
connection.disconnect and inputstream.close then inside finally (needs some extra try-catch, but thats the safest way)